create-module.sh

```diff
@@ -264,12 +264,13 @@ cat > $readme << EOF
 EOF
 
 # Adds module dependency and export for React and CSS
+node "utils/create-module.js" "$name" "react";
```

Comment:
This is the idea of splitting it up into 2 different calls - one for the React kit and one for the CSS kit

Reply:
Right I gotcha 👍 so if you look at my latest though, I discovered all of this as to be done by setting a boolean and running both react / css nodejs > `yarn install` completes. Kinda makes sense after realizing it haha

Reply:
I just prefer an explicit separate call vs a boolean:

```ts
// do two different things
doThingOne()
doThingTwo()

// vs
const andDoThingTwo = true
doThingOne(andDoThingTwo)
```

Reply:
I could understand a boolean argument for some type of behavior modifier, much like how we think about Props in a UI Component:

```ts
doSomething({ animated: false })
```

It just feels odd to do different things with a boolean parameter. I understand that the `yarn create-module` does it this way by creating a React component no matter what and optionally allowing you to also make a CSS module. I don't think the underlying function calls should do the same thing.